site/en/blog/_example/index.md

```diff
@@ -0,0 +1,60 @@
+---
+# Required
+title: An example blog post
+
+# Required
+description: Describe your example post with a couple sentences.
+
+# Required
+layout: 'layouts/blog-post.njk'
+
+# Required
+date: 2021-01-27
+
+# Optional
+# Include an updated date when you update your post
+updated: 2021-01-28
+
+# Optional
+# How to add a new author
+# https://developer.chrome.com/docs/handbook/how-to/add-an-author/
+authors:
+  - samthorogood
+  - robdodson
+  - kaycebasques
+
+# Optional
+# How to a new tag
+# https://developer.chrome.com/docs/handbook/how-to/add-a-tag/
+tags:
+  - privacy
+  - security
+
+# Optional
+# Hero images should be at least 1920 x 960
+# https://developer.chrome.com/docs/handbook/how-to/add-media/
+hero: 'image/BrQidfK9jaQyIHwdw91aVpkPiib2/EnMzOm0mBytBA3AzlCG6.png'
+
+# Optional (if there is no hero image)
```

Comment:
This is ambigious, could it instead say "Required if there is a hero image"—it makes no sense without a hero image.